app/services/admin/api/cms/template_service.rb

```diff
@@ -114,6 +115,22 @@ def attach_layout
     end
   end
 
+  class Destroy
+    include Callable
+
+    def initialize(template)
+      @template = template
+    end
+
+    attr_reader :template
+
+    def call
+      raise DeleteBuiltinError, "Builtin resources can't be deleted" unless template.respond_to?(:destroy)
```

Comment:
I think it would be better to use "Built-in", as it seems that this is the "human-readable" way used in other places, e.g.
https://github.com/3scale/porta/blob/d71951a27032bb027b72b772015c77d0705b0099/app/models/cms/template.rb#L141
https://github.com/3scale/porta/blob/d71951a27032bb027b72b772015c77d0705b0099/config/locales/en.yml#L1015-L1016

Reply:
Done!